### PR TITLE
EMPRO Resources Menu Item for Substudy Staff and Clinicians

### PR DESCRIPTION
**Pull request by achen2401**
Address https://jira.movember.com/browse/TN-2834
present EMPRO Resources in the burger menu for Sub-Study Staff and Clinicians

Changes include:
- add EMRPO Resources menu item to portal wrapper burger menu
- restrict access to the burger menu item to staff/staff admin/clinician users
- restrict access to the burger menu item only to users in the sub-study
- add frontend global helper function to easily access the research studies that the user is in

### CODE REVIEW

File: portal/templates/portal_wrapper.html [hunk 1]
```diff
@@ -89,6 +89,9 @@
                             {% if user and user.has_role(ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
                             <li class="resources-menu-item"><a href="{{PORTAL}}/resources">{{ _("Resources") }}</a></li>
                             {% endif %}
+                            {% if user and user.has_role(ROLE.CLINICIAN.value, ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
+                                <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPROS Resources") }}</a></li>
```
Comment (ivan-c) [inline portal/templates/portal_wrapper.html:93]: Should this be `EMPRO` not `EMPRO`**S**?
```suggestion
                                <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPRO Resources") }}</a></li>
```

File: portal/templates/portal_wrapper.html [hunk 2]
```diff
@@ -142,6 +145,9 @@
                     {% if user and user.has_role(ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
                         <li class="resources-menu-item"><a href="{{PORTAL}}/resources">{{ _("Resources") }}</a></li>
                     {% endif %}
+                    {% if user and user.has_role(ROLE.CLINICIAN.value, ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
+                        <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPROS Resources") }}</a></li>
```
Comment (ivan-c) [inline portal/templates/portal_wrapper.html:149]:
```suggestion
                        <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPRO Resources") }}</a></li>
```

**Review by ivan-c — APPROVED** [review]
Saw a minor spelling discrepancy, but wasn't sure if it was intentional. Looks good otherwise!

File: portal/templates/portal_wrapper.html (hunk 1 above)
Comment (achen2401) [reply portal/templates/portal_wrapper.html]: thank you for catching that, Ivan! Fixed

**Pull request merged**